src/Sulu/Component/Content/Compat/Structure/StructureBridge.php

```diff
@@ -124,7 +120,7 @@ public function getWebspaceKey()
      */
     public function getUuid()
     {
-        return $this->getDocument()->getUuid();
+        return $this->getDocument()->getUuid($this->document);
```

Comment:
And the method doesn't take an argument as you can see in https://github.com/sulu-io/sulu/blob/ca0f510f63386763564b99bafbfd45bad5e7a76d/src/Sulu/Bundle/ContentBundle/Document/BasePageDocument.php#L402